### PR TITLE
Debian 10 detection

### PR DESCRIPTION
**Pull request by kiemlicz**
removing old logic that installs debian 9 on debian 10

### What does this PR do?
Detects Debian 10 for further Salt's repo setup

### What issues does this PR fix or reference?
https://github.com/saltstack/salt-bootstrap/issues/1374

### Previous Behavior
debian 9 packages installed on debian 10

### New Behavior
debian 10 packages installed on debian 10

### CODE REVIEW

**Review by vutny — APPROVED** [review]
Good job, @kiemlicz 
This should enable correct installation on Debian 10 stable.

@s0undt3ch Please take a look, thanks.

**Review by s0undt3ch — APPROVED** [review]

**Comment by kiemlicz** [comment]
I assume that someone will merge it when possible (I don't seem to have write access to repo)?

**Comment by s0undt3ch** [comment]
Yes, I'll merge it once I'm happy with our CI build.

**Comment by johnkeates** [comment]
CI seems stuck :(

**Comment by johnkeates** [comment]
@s0undt3ch any idea why CI stays stuck?

**Comment by s0undt3ch** [comment]
Sorry it took so long @johnkeates. Thank You!

**Pull request merged**
